server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -337,11 +347,19 @@ public void execute(final String[] args) {
 
     cl.parse(args);
 
-    if (opts.help || cl.getParsedCommand() == null) {
+    if (cl.getParsedCommand() == null) {
       cl.usage();
       return;
     }
 
+    if (opts.help || listInstancesOpts.help || pingCommand.help || checkTabletsCommand.help
+        || stopOpts.help || dumpConfigCommand.help || volumesCommand.help
+        || verifyTabletAssignmentsOpts.help || deleteZooInstOpts.help || restoreZooOpts.help
+        || fateOpsCommand.help || tServerLocksOpts.help || serviceStatusCommandOpts.help) {
+      cl.getCommands().get(cl.getParsedCommand()).usage();
+      return;
+    }
```

Comment:
I tried running this locally and it was really fast to only print help.  I tried running `accumulo admin stopAll -h` and it did not print help and actually ran stopped stuff instead.  I think that was because the command was not in the if stmt.   Tried automatically finding stuff instead of the if stmt and that seemed to fix it.

```suggestion
    for(var command : cl.getCommands().entrySet()){
      var objects = command.getValue().getObjects();
      for(var obj : objects) {
        if(obj instanceof SubCommandOpts && ((SubCommandOpts)obj).help) {
          command.getValue().usage();
          return;
        }
      }
    }
```